test/NeoSharp.Core.Test/SmartContracts/UtContractParameterType.cs

```diff
@@ -0,0 +1,129 @@
+            Assert.AreEqual(scriptBuilder.Length, 1);
+            scriptBuilder.Clear();
+
+            //public IntegerContractParameter(short value) : base(ContractParameterType.Integer, value) { }
+            short shortVariable = 1;
+            integerContractParameter = new IntegerContractParameter(shortVariable);
+            integerContractParameter.PushIntoScriptBuilder(scriptBuilder);
+            Assert.AreEqual(scriptBuilder.Length, 1);
+            scriptBuilder.Clear();
+
+            //public IntegerContractParameter(ushort value) : base(ContractParameterType.Integer, value) { }
+            ushort uShortVariable = 1;
+            integerContractParameter = new IntegerContractParameter(uShortVariable);
+            integerContractParameter.PushIntoScriptBuilder(scriptBuilder);
+            Assert.AreEqual(scriptBuilder.Length, 1);
+            scriptBuilder.Clear();
+
+            //public IntegerContractParameter(long value) : base(ContractParameterType.Integer, value) { }
+            long longVariable = 1;
+            integerContractParameter = new IntegerContractParameter(longVariable);
+            integerContractParameter.PushIntoScriptBuilder(scriptBuilder);
+            Assert.AreEqual(scriptBuilder.Length, 1);
+            scriptBuilder.Clear();
+
+            //public IntegerContractParameter(ulong value) : base(ContractParameterType.Integer, value) { }
+            ulong uLongVariable = 1;
+            integerContractParameter = new IntegerContractParameter(uLongVariable);
+            integerContractParameter.PushIntoScriptBuilder(scriptBuilder);
+            Assert.AreEqual(scriptBuilder.Length, 1);
+            scriptBuilder.Clear();
+
+            //public IntegerContractParameter(byte value) : base(ContractParameterType.Integer, value) { }
+            byte byteVariable = 1;
+            integerContractParameter = new IntegerContractParameter(byteVariable);
+            integerContractParameter.PushIntoScriptBuilder(scriptBuilder);
+            Assert.AreEqual(scriptBuilder.Length, 1);
+            scriptBuilder.Clear();
+
+            //public IntegerContractParameter(sbyte value) : base(ContractParameterType.Integer, value) { }
+            sbyte signedByteVariable = 1;
+            integerContractParameter = new IntegerContractParameter(signedByteVariable);
+            integerContractParameter.PushIntoScriptBuilder(scriptBuilder);
+            Assert.AreEqual(scriptBuilder.Length, 1);
+            scriptBuilder.Clear();
+
+            //public IntegerContractParameter(BigInteger value) : base(ContractParameterType.Integer, value) { }
+            BigInteger bigIntegerVariable = new BigInteger(1);
+            integerContractParameter = new IntegerContractParameter(bigIntegerVariable);
+            integerContractParameter.PushIntoScriptBuilder(scriptBuilder);
+            Assert.AreEqual(scriptBuilder.Length, 1);
+            scriptBuilder.Clear();
```

Comment:
Split this test out into multiple test methods. And separate test files for each class.